modular_chomp/code/modules/mob/living/silicon/robot/sprites/civilian.dm

```diff
@@ -23,22 +23,22 @@
 /datum/robot_sprite/dogborg/tall/service/dullahanservice
```

Comment:
We have multiple of the same type, move the DMI and offset to a parent path.

Reply:
where should i move the dmi? i dont understand this part